packaging/Makefile_packaging.mk

```diff
@@ -68,6 +68,7 @@ endef
 
 define install_repos
 	for repo in $($(DISTRO_BASE)_PR_REPOS)                              \
+	            $($(DISTRO_BASE)_LOCAL_REPOS)                           \
```

Comment:
You don't want to run these through this loop, but rather do the:

```bash
	rm -f /etc/yum.repos.d/*"${DAOS_STACK_LOCAL_REPO//\//_}"		
￼	yum-config-manager --add-repo="$REPOSITORY_URL"/"$DAOS_STACK_LOCAL_REPO"		
￼	echo "gpgcheck = False" >> /etc/yum.repos.d/*"${DAOS_STACK_LOCAL_REPO//\//_}".repo
```

process with them.  But just be aware that in the above, `DAOS_STACK_LOCAL_REPO` is just the path portion of the URL and `$($(DISTRO_BASE)_LOCAL_REPOS)` will be entire URLs, so you will have to extract the path portion out.